client/src/pages/auth/Login.tsx

```diff
@@ -1,31 +1,90 @@
-import { useRef } from 'react';
+import { useRef, useState } from 'react';
 import { Link } from 'react-router-dom';
 import InputBox from '@/pages/auth/InputBox';
 import Button from '@/components/Button';
+import type { ResponseWithoutData } from "@/fetches/common/response.type";
+import { server } from "@/fetches/common/axios";
 
 function Login() {
   const emailInputRef = useRef<HTMLInputElement | null>(null);
   const pwdInputRef = useRef<HTMLInputElement | null>(null);
 
-  const handleLogin = () => {
-    const userEmail = emailInputRef.current?.value;
-    const userPwd = pwdInputRef.current?.value;
-    if (userEmail == '') {
-      alert('아이디를 입력해주세요');
-    } else if (userPwd == '') {
-      alert('비밀번호를 입력해주세요');
-    } else {
-      // 서버에 로그인 요청
+  const [isWrongEmail, setWrongEmail] = useState(false);
+  const [emailErr, setEmailErr] = useState("");
+
+  const [isWrongPwd, setWrongPwd] = useState(false);
+  const [pwdErr, setPwdErr] = useState("");
+
+  const handleLogin = async () => {
+    const userEmail: string = emailInputRef.current?.value || '';
+    const userPwd: string = pwdInputRef.current?.value || '';
```

Comment:
```suggestion
    const userEmail: string = emailInputRef.current?.value ?? '';
    const userPwd: string = pwdInputRef.current?.value ?? '';
```
`??` 연산자를 사용하는 것이 좋을 것 같아요!